From linux-usb-owner@vger.kernel.org Sat Mar  8 00:13:17 2008
From: "Robert P. J. Day" <rpjday@crashcourse.ca>
Date: Sat, 8 Mar 2008 03:12:44 -0500 (EST)
Subject: USB: Remove EXPERIMENTAL from dynamic USB minor allocation.
To: linux-usb@vger.kernel.org
Message-ID: <alpine.LFD.1.00.0803080307060.7171@localhost.localdomain>

Since this USB feature seems non-experimental, remove that dependency.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/Kconfig |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/core/Kconfig
+++ b/drivers/usb/core/Kconfig
@@ -76,8 +76,8 @@ config USB_DEVICE_CLASS
 	    NAME="bus/usb/$env{BUSNUM}/$env{DEVNUM}", MODE="0644"
 
 config USB_DYNAMIC_MINORS
-	bool "Dynamic USB minor allocation (EXPERIMENTAL)"
-	depends on USB && EXPERIMENTAL
+	bool "Dynamic USB minor allocation"
+	depends on USB
 	help
 	  If you say Y here, the USB subsystem will use dynamic minor
 	  allocation for any device that uses the USB major number.
